lasagne/layers/__init__.py

```diff
@@ -8,4 +8,5 @@
 from .shape import *
 from .merge import *
 from .normalization import *
+from .prelu import *
```

Comment:
I agree, that would make more sense. No idea what to call it though. `lasagne.layers.nonlinearities` would be easily confusable with `lasagne.nonlinearities`, so that's probably a bad idea.